3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune/PCAWrapper.cs

```diff
@@ -29,27 +29,29 @@ public class PCAWrapper
         /// <summary>
         /// The authority for the MSAL PublicClientApplication. Sign in will use this URL.
         /// </summary>
-        private const string _authority = "https://login.microsoftonline.com/organizations";
+        private const string _authority = "https://login.microsoftonline.com/organizations"; // TODO - Optionally replace organizations with your tenant ID or name
 
-        // ClientID of the application in (msidlab4.com)
-        private const string ClientId = "94996db4-9d57-422d-a707-84a1328a3cb8"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml
-
-        // TenantID of the organization (msidlab4.com)
-        private const string TenantId = "f645ad92-e38d-4d1a-b510-d1b09a74a8ca"; // TODO - Replace with your TenantID. And also replace in the AndroidManifest.xml
+        // ClientID of the application
+        private const string ClientId = "TODO <Your ClientID>"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml
 
         private string[] clientCapabilities = { "ProtApp" }; // It is must to have these capabilities
 
+        void MyLoggingMethod(LogLevel level, string message, bool containsPii)
+        {
+            Console.WriteLine($"MSAL .NET {level}: {message}");
+        }
+
         // private constructor for singleton
         private PCAWrapper()
         {
             // Create PCA once. Make sure that all the config parameters below are passed
             // ClientCapabilities - must have ProtApp
             PCA = PublicClientApplicationBuilder
                                         .Create(ClientId)
+                                        .WithClientCapabilities(clientCapabilities)
+                                        .WithLogging(MyLoggingMethod, LogLevel.Verbose, enablePiiLogging: true, enableDefaultPlatformLogging: false)
```

Comment:
Do you want to keep this logic to enable verbose/PII logging to the console?

Reply:
Removed